React/React hooks.md

```diff
@@ -0,0 +1,754 @@
+### React hooks
```

Comment:
You should fix the headings. The title will be shown automatically based on the file name. All other headings should start from h2